android/capacitor/src/main/java/com/getcapacitor/plugin/util/HttpRequestHandler.java

```diff
@@ -398,6 +398,19 @@ public static JSObject request(PluginCall call, String httpMethod, Bridge bridge
 
         boolean isHttpMutate = method.equals("DELETE") || method.equals("PATCH") || method.equals("POST") || method.equals("PUT");
 
+        // a workaround for the following android web view issue:
+        // https://issues.chromium.org/issues/40450316
+        // x-cap-user-agent contains the user agent set in JavaScript
+        String userAgentValue = headers.getString("x-cap-user-agent");
+        if (userAgentValue != null && !userAgentValue.isEmpty()) {
+            headers.put("User-Agent", userAgentValue);
+        }
+        headers.remove("x-cap-user-agent");
+
+        if (overriddenUserAgent != null && !headers.has("User-Agent") && !headers.has("user-agent")) {
```

Comment:
```suggestion
        if (!headers.has("User-Agent") && !headers.has("user-agent")) {
```
Not sure if you didn't understand what I meant, I said that in the first commit you were doing a `overriddenUserAgent != null` check and you removed it.
One option was to put the check back, and the other option was to remove the variable and just put the `bridge.getConfig().getOverriddenUserAgentString()` in the put directly since it didn't really matter if it was null.
Looks like you did both options, but since you removed the `overriddenUserAgent` variable, now the code doesn't compile because you added the null check for it.

Reply:
Yup, sorry, completely missed removing that first part of the statement.